client/homebrew/pages/vaultPage/vaultPage.jsx

```diff
@@ -248,6 +263,43 @@ const VaultPage = (props)=>{
 		</div>
 	);
 
+	const renderSortOption = ( optionTitle, optionValue)=>{
+		const oppositeDir = dirState === 'asc' ? 'desc' : 'asc';
+
+		return (
+			<div
+			className={`sort-option ${
+                sortState === optionValue ? `active ${oppositeDir}` : ''
```

Comment:
I don't see that `acc` or `desc` in the className here has any effect. It's not mentioned in the `.less` file at all.
```suggestion
                sortState === optionValue ? `active` : ''
```